pandas/tests/io/test_parquet.py

```diff
@@ -931,6 +931,17 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    @td.skip_if_no("pyarrow", min_version="4.0.0")
+    def test_list_column_results_in_compliant_parquet(self, pa):
+        # https://github.com/pandas-dev/pandas/issues/43689
+        df = pd.DataFrame({"a": [[1], [2]]})
+
+        with tm.ensure_clean() as path:
+            df.to_parquet(path, pa)
+            result = pyarrow.parquet.read_table(path)
+
+        assert str(result.schema.field_by_name("a").type) == "list<element: int64>"
```

Comment:
use `check_round_trip(df, pa)` as this will assert the results

Reply:
That isn't what I'm testing though... I'm testing that the Parquet that is written is correct... 

ie.  "list<element: int64>" not  "list<item: int64>"

Reply:
https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#nested-types

Reply:
https://github.com/apache/arrow/pull/9489

Reply:
well if the serialize & de-serialize works then i am not sure i understand the problem. you can certainly assert the meta-data if you want. the most important point is that you *need* this option to have correct results yes?

Reply:
The problem is that `to_parquet` doesn't write Parquet compliant data. 

What if I want to load data from Pandas into a different language?

Reply:
Here is an example of a place where this is an issue: https://github.com/googleapis/python-bigquery/pull/980

Reply:
In summary: yes, `use_compliant_nested_type` is needed in order to output the correct, compliant Parquet format.